src/components/UsersSearch.vue

```diff
@@ -86,21 +134,32 @@ export default {
       },
       {
         align: 'left',
-        text: 'Roles',
+        text: 'Role',
         value: 'roles'
       },
+      {
+        align: 'right',
+        value: 'indicators',
+        width: '5%'
+      },
       {
         align: 'right',
         value: 'icons',
         width: '5%'
       }
     ],
 
+    optionItems: ['Deactivated'],
```

Comment:
I think we should add the ability to filter to show _only is_active users_ as well.

Reply:
note we decided today not to call users "disabled"